### PR TITLE
[NEW][10.0] POS partner firstname support

### PR DESCRIPTION
**Pull request by robyf70**
Add the support of First/Last name in the POS using the partner_firstname module

### CODE REVIEW

File: pos_partner_firstname/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.model
+    def create_from_ui(self, partner):
+        if 'is_company' in partner:
+            partner['is_company'] = partner['is_company'] == 'true'
+        if 'electronic_invoice_subjected' in partner:
```
Comment (eLBati) [inline pos_partner_firstname/models/res_partner.py]: This part should stay in the Italian module

**Review by eLBati — CHANGES_REQUESTED** [review]
For the rest, :+1:

File: pos_partner_firstname/models/res_partner.py (hunk 1 above)
Comment (robyf70) [reply pos_partner_firstname/models/res_partner.py]: @eLBati Right! But only the part related to ```electronic_invoice_subjected``` need to be removed, the rest is necessary.

File: pos_partner_firstname/static/src/js/screens.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+odoo.define('pos_partner_firstname.screens', function (require) {
```
Comment (eLBati) [inline pos_partner_firstname/static/src/js/screens.js:1]: `pos_partner_firstname.screens` > `pos_partner_firstname.models`

**Review by eLBati — CHANGES_REQUESTED** [review]

File: pos_partner_firstname/static/src/js/screens.js (hunk 2 above)
Comment (robyf70) [reply pos_partner_firstname/static/src/js/screens.js:1]: Ah! Right! I thought I did already ... anyway ... committed the fix :)

**Review by eLBati — APPROVED** [review]
Thanks

**Comment by robyf70** [comment]
@legalsylvain Can you please review this PR?

**Comment by legalsylvain** [comment]
Sorry, no time for the time being. I'll do in after the hollidays. feel free to ping me if I forget !

**Comment by eLBati** [comment]
@legalsylvain @robyf70 as "beta" module, I think it could be merged with 1 review

**Comment by legalsylvain** [comment]
make sense.

/ocabot merge

**Pull request merged**